src/functionalTest/groovy/com/autonomousapps/jvm/SimilarAdviceSpec.groovy

```diff
@@ -14,12 +14,17 @@ final class SimilarAdviceSpec extends AbstractJvmSpec {
     gradleProject = project.gradleProject
 
     when: 'We run the same tasks on two different projects, with build cache enabled'
-    build(gradleVersion, gradleProject.rootDir, 'lib1:aggregateAdvice', '--build-cache')
-    build(gradleVersion, gradleProject.rootDir, 'lib2:aggregateAdvice', '--build-cache')
+    def task = 'aggregateAdvice'
+    if (!isV1()) {
+      task = 'filterAdvice'
+      build(gradleVersion, gradleProject.rootDir, 'buildHealth')
```

Comment:
Take a stronger look at this. Does this test make sense any more? Delete for v2?